cost-analyzer/templates/_helpers.tpl

```diff
@@ -1192,12 +1192,10 @@ Begin Kubecost 2.0 templates
       readOnly: false
     - name: tmp
       mountPath: /tmp
-    {{- range $key := .Values.kubecostModel.plugins.enabledPlugins }}
     - mountPath: {{ $.Values.kubecostModel.plugins.folder }}/config
       name: plugins-config
```

Comment:
the range wouldn't have worked. I'm assuming I broke this when we worked on this last friday